src/core/creative.js

```diff
@@ -0,0 +1,143 @@
+import { SortedListItem } from '../util/sorted-list-item'
+import { UniversalAdId } from './universal-ad-id'
+
+/**
+ * A creative.
+ *
+ * @class Creative
+ * @extends SortedListItem
+ * @author Tim De Pauw <tim.depauw@zentrick.com>
+ * @copyright © 2018 Zentrick nv
+ */
+export class Creative extends SortedListItem {
+  constructor () {
+    super()
+    this._universalAdId = new UniversalAdId()
+    this._creativeExtensions = []
+  }
+
+  // Attribute(s).
+
+  /**
+   * The ID of this creative.
+   *
+   * Bounded: 1
+   *
+   * @type {string}
+   */
+  get id () {
+    return this._id
+  }
+
+  set id (value) {
+    this._id = value
+  }
+
+  /**
+   * The ID of the {@link Ad} with which this creative is served.
+   *
+   * Bounded: 0-1 if VAST 4.0, 1 otherwise
+   *
+   * @type {string}
+   */
+  get adID () {
+    return this._adID
+  }
+
+  set adID (value) {
+    this._adID = value
+  }
+
+  /**
+   * The string that identifies an API that is needed to execute this creative.
+   *
+   * Bounded: 1
+   *
+   * @type {string}
+   * @since VAST 4.0
+   */
+  get apiFramework () {
+    return this._apiFramework
+  }
+
+  set apiFramework (value) {
+    this._apiFramework = value
+  }
+
+  // Sub-element(s).
+
+  /**
+   * The unique creative identifier for this creative.
+   *
+   * Bounded: 1
+   *
+   * @type {UniversalAdId}
+   * @readonly
+   * @since VAST 4.0
+   */
+  get universalAdId () {
+    return this._universalAdId
+  }
+
+  /**
+   * The creative extensions for this creative. `<CreativeExtensions>` as no
+   * attribute and is not recorded.
+   *
+   * Bounded: 0+
```

Comment:
So basically unbounded. Not sure if we need to make it explicit then. This ties into the whole discussion about validation that I really don't want to take too far for this already big PR.